qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -18,6 +38,12 @@ You can download an example sample information file and prep information file fr
 Sample information file
 -----------------------
 
+The *sample information file* will define the biological context of each
+sample, with categories like ``sample_type``, ``treatment``, ``timepoint``,
```

Comment:
Remove `timepoint` from this list to avoid confusion (the correct name is `collection_timestamp` as described below.